### ISSUE TITLE
Document how to use scrollTop() properly

### ISSUE THREAD

**Issue by jzaefferer** [jquery/api.jquery.com#417]
Via discussion in [#14515](http://bugs.jquery.com/ticket/14515).

This should be added to [scrollTop()](https://github.com/jquery/api.jquery.com/blob/master/entries/scrollTop.xml):

``` md
Note that scrollTop can be used to read, but not write, the document scroll position. To scroll the document, animated or not, you have to call `scrollTop()` on the body element:

<pre><code>
// Without animation
$( document.body ).scrollTop( 500 );

// With animation
$( document.body ).animate({
  scrollTop: 500
});
</code></pre>

Any examples elsewhere that suggest $( "html, body" ).animate({ scrollTop: ... }); are wrong, since this will cause the animation callback to get triggered twice.
```

**Comment by arthurvr** [comment]
> I'll try to write something by the weekend.

@leobalter Still interested in taking this? Or did you already do?

**Comment by leobalter** [comment]
I failed doing this. I may try it again but it shouldn't hold anyone else to fix it.

**Comment by jzaefferer** [comment]
@arthurvr interested in taking this over? Or finding someone to write the docs? Sad to see this still open more than a year later...

**Comment by arthurvr** [comment]
> interested in taking this over? Or finding someone to write the docs? Sad to see this still open more than a year later...

That's true for basically ever ticket open here :( One day I'll get to for sure, but time just limits me.

**Comment by arthurvr** [comment]
@jzaefferer Although I do definitely agree that many things in this repo are taking way too long. There are still APIs from back in 2012 undocumented, just because we lack the time and the resources.

**Comment by dmethvin** [comment]
@arthurvr the important thing is that we are making progress, thanks to you! This issue in particular seemed to get lots of people suggesting changes without a clear consensus on what to do. I think that's why it sat around so long, it is tedious to sort through these kinds of tickets and there is always concern you'll upset someone by ignoring their concerns. So go for the low-hanging fruit if it's easier.

**Comment by jzaefferer** [comment]
Here's a good explanation of the underlying mess, along with a future-proof solution called `document.scrollingElement`: https://dev.opera.com/articles/fixing-the-scrolltop-bug/

If I read that correctly, we should document how to animate `document.scrollingElement`, along with a recommendation to load [the polyfill](https://github.com/mathiasbynens/document.scrollingElement).
